bin/pic-configure

```diff
@@ -165,6 +165,14 @@ if [ -f "$extension_param/cmakeFlags" ] ; then
         echo "       Is the file executable? (chmod u+x cmakeFlags)" >&2
         exit 2
     fi
+    # save the cmakeFlags setup number for future reference
+    if [ -f "$extension_param/cmakeFlagsSetup" ] ; then
+      rm $extension_param/cmakeFlagsSetup
```

Comment:
Just to be sure: The file is only deleted, if the option `-t` is used, isn't it?

Reply:
No, when `-t` is not used it would make the same thing for the default setup=0.